tests/test_pipeline.py

```diff
@@ -33,6 +44,36 @@
 _RANDOM_STATE = 67056
 
 
+def _get_rf_regressor() -> Pipeline:
+    """Get a morgan + physchem + random forest pipeline.
+
+    To make the run extra slow, a TauTomerCanonicalizer is added.
+
+    Returns
+    -------
+    Pipeline
+        A pipeline with a morgan fingerprint, physchem descriptors, and a random forest
+    """
+    smi2mol = SmilesToMol()
+    tautomerizer = TautomerCanonicalizer()
+    mol2concat = MolToConcatenatedVector(
+        [
+            ("mol2morgan", MolToMorganFP(radius=2, n_bits=2048)),
+            ("mol2physchem", MolToRDKitPhysChem()),
+        ]
+    )
+    rf = RandomForestRegressor(random_state=42, n_jobs=-1)
```

Comment:
don't do unit tests that suddenly use multiprocessing. This leads to unpredictable resource allocation. Set `n_jobs=1` (and I know that this is overwritten by the Pipeline's n_jobs argument)